dbhydra/dbhydra_core.py

```diff
@@ -763,15 +871,19 @@ def modify_column(self, column_name, new_column_type):
 class MongoTable():
     def __init__(self, db, name, columns=[], types=[]):
         self.name = name
-        self.db = db
+        self.db1 = db
```

Comment:
why doesnt mongotable inherit from abstract table?
if we don't inherit willingly then please add comment to code why we don't do that

Reply:
This is ok, Mongo will have different inheritance structure as it is a NoSQL (many methods are different)